tests/test_exceptions.py

```diff
@@ -0,0 +1,14 @@
+"""Trivial tests on our exceptions."""
+
+import pytest
+
+
+def test_import_old_serialize_exceptions():
+    """Test importing from pulser.json.exceptions.
+
+    This should trigger a warning.
+    """
+    with pytest.warns(match="module pulser.json.exceptions is deprecated"):
```

Comment:
Can you check it's a `DeprecationWarning`?
```suggestion
    with pytest.warns(DeprecationWarning, match="module pulser.json.exceptions is deprecated"):
```